bokeh/sampledata/us_holidays.py

```diff
@@ -4,9 +4,24 @@
 #
 # The full license is in the file LICENSE.txt, distributed with this software.
 #-----------------------------------------------------------------------------
-'''
+''' A calendar file of US Holidays from Mozilla provided by `icalendar`_.
```

Comment:
```suggestion
''' Calendar file of US Holidays from Mozilla provided by `icalendar`_.
```

My impression is that most of those docstrings don't start with an (indefinite) article.